administrator/components/com_templates/src/Helper/TemplateHelper.php

```diff
@@ -98,7 +98,7 @@ public static function canUpload($file, $err = '')
 
         $allowable = array_merge($imageTypes, $sourceTypes, $fontTypes, $archiveTypes);
 
-        if ($format == '' || $format == false || (!\in_array($format, $allowable))) {
+        if ($format == '' || !$format || (!\in_array($format, $allowable))) {
```

Comment:
$format is a string, so the original code compare $format with false is useless, and could be removed
```suggestion
        if ($format === '' || !\in_array($format, $allowable)) {
```